ingest/workflow/snakemake_rules/fetch_sequences.smk

```diff
@@ -0,0 +1,49 @@
+"""
+This part of the workflow handles fetching sequences from various sources.
+Uses `config.sources` to determine which sequences to include in final output.
+
+Currently only fetches sequences from GenBank, but other sources can be
+defined in the config. If adding other sources, add a new rule upstream
+of rule `fetch_all_sequences` to create the file `data/{source}.ndjson` or the
+file must exist as a static file in the repo.
+
+Produces final output as
+
+    sequences_ndjson = "data/sequences.ndjson"
+
+"""
+
+def download_serotype(w):
+    serotype = {
+        'all': '12637',
+        'denv1': '11053',
+        'denv2': '11060',
+        'denv3': '11069',
+        'denv4': '11070'
+    }
+    return serotype[w.serotype]
```

Comment:
To be consistent with `_get_all_sources` and avoid single char variable names (which are maybe appropriate for very contextual lambdas, but not in functions defined elsewhere):

```suggestion
def download_serotype(wildcards):
    serotype = {
        'all': '12637',
        'denv1': '11053',
        'denv2': '11060',
        'denv3': '11069',
        'denv4': '11070'
    }
    return serotype[wildcards.serotype]
```